src/API/IJobsRepository.cs

```diff
@@ -39,6 +39,14 @@ public interface IJobRepository
         /// <param name="job"><see cref="Nvidia.Clara.DicomAdapter.API.InferenceJob" /></param>
         Task Add(InferenceJob job);
 
+        /// <summary>
+        /// Queues a new job for submission and disable EF change tracking.
+        /// <c>New</c> makes a copy of the instances to a temporary location that is to be
+        /// uploaded by the `Nvidia.Clara.DicomAdapter.Server.Services.Jobs.JobSubmissionService`.
+        /// </summary>
+        /// <param name="job"><see cref="Nvidia.Clara.DicomAdapter.API.InferenceJob" /></param>
+        Task AddWithoutTracking(InferenceJob job);
```

Comment:
Should the Add method simply be updated to include tracking flag param? The implementation class below ends up calling the AddJob method with the flag param, wonder why having two separate method at the I/F level.

Reply:
Updated